src/components/AdditionalInfo/AdditionalInfo.jsx

```diff
@@ -3,21 +3,36 @@ import PropTypes from 'prop-types';
 import _ from 'lodash';
 import classNames from 'classnames';
 import ExpandingGroup from '../ExpandingGroup/ExpandingGroup';
+import dispatchAnalyticsEvent from '../../helpers/analytics';
 
 export default class AdditionalInfo extends React.Component {
   constructor(props) {
     super(props);
     this.expandedContentId = _.uniqueId('tooltip-');
     this.state = { open: false };
+
+    this.toggle = this.toggle.bind(this);
   }
 
-  toggle = () => {
+  toggle() {
     this.setState({ open: !this.state.open });
+
+    // Conditionally track the event.
```

Comment:
So this will work fine since `this.setState()` is asynchronous, but for a little better readability, I'd recommend putting the tracking code before the `setState` call. That way it's clear that the `action: !this.state.open...` refers to the state _before_ it was switched.